.github/workflows/basic.yml

```diff
@@ -14,8 +14,9 @@ jobs:
         command:
           - pytest
           - DIRAC_USE_M2CRYPTO=Yes pytest
-          - pytest Core/Security/test
-          - DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test
+          # Security tests are flakey due to reference counting bugs in pyGSI/src/crypto/asn1.c
+          - pytest Core/Security/test || (echo "Retrying..." pytest Core/Security/test) || (echo "Retrying again..." pytest Core/Security/test)
+          - DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test || (echo "Retrying..." DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test) || (echo "Retrying again..." DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test)
```

Comment:
I don't know the details, but... really? is this the only thing we can do?
And BTW does it happen only with the M2CRYPTO flag on? And, why only now?

Reply:
It only happens for the security tests but affects running both with and without the M2CRYPTO flag (the tests are comparing PyGSI and M2CRYPTO). @chaen Mentioned this has been an issue in the past when the tests were on travis. I suspect the issue depends on the state of the hypervisor and that is the reason we can't reproduce it locally.

Yesterday I managed to get some core dumps of the actual crash. It happens during garbage collection and I think the correct fix would be to fix the reference counting in pyGSI but, as pyGSI will be dropped soon, it doesn't seem like a good use of time. The test can't be marked as flaky and retried within `pytest` as it's a segfault. Setting `PYTHONVERBOSE=1`appears to avoid the crash but causes a lot of output. I think this is the best solution for now unless you have any other ideas?